express/__init__.py

```diff
@@ -29,23 +28,18 @@ def __init__(self, parser_name=None, *args, **kwargs):
     def _get_parser_class(self, parser_name):
         """
         Returns parser class for a given parser name.
-
```

Comment:
Here and below: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Empty lines should be kept